### DISCUSSION TITLE
Is there a way to access search history?

### DISCUSSION THREAD

**Discussion by ni554n** [gdh1995/vimium-c#1070] [Q&A]
Sometimes I need to search for something in youtube and then in google. I want to avoid typing the keyword twice.

**Comment by gdh1995** [comment]
You may try the command named `searchInAnother`: `map xxx searchInAnother keyword="g" reuse="new-fg"` will make Vimium C parse URL of a current tab, get its query words and then search it using the custom search engine named `g`.

To make Vimium C find query words successfully, your configuration of `custom search engine` should include an item for youtube, like:
```
yt: https://www.youtube.com/results?search_query=$s  blank=https://www.youtube.com/ YouTube
```
